src/chart/helper/EffectLine.js

```diff
@@ -177,6 +177,7 @@ effectLineProto.updateSymbolPosition = function (symbol) {
     var cp1 = symbol.__cp1;
     var t = symbol.__t;
     var pos = symbol.position;
+    var lastPos = Array.from(pos);
```

Comment:
Please use `Array.prototype.slice.call(pos)` instead of `Array.from` because it's not compatible in IE.